packages/devtools_app/test/chart_test.dart

```diff
@@ -0,0 +1,931 @@
+import 'dart:io';
+
+@TestOn('vm')
+import 'package:devtools_app/src/charts/chart_controller.dart';
+import 'package:devtools_app/src/charts/chart_trace.dart';
+import 'package:devtools_app/src/charts/chart.dart';
+import 'package:devtools_shared/devtools_shared.dart';
+import 'package:devtools_testing/support/memory_test_data.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'support/wrappers.dart';
+
+void main() {
+  const windowSize = Size(2225.0, 1000.0);
+
+  final nowDT = DateTime.now();
+  final isUSLocalePST = Platform.localeName == 'en_US' &&
```

Comment:
can we force the  Platform and time zone instead of having the tests quietly skip if not run in that zone? E.g. it would be a shame if these tests were skipped on the bots.

Reply:
I'll take a look but the tests ran everywhere except one bot.